ios-release-build.md

```diff
@@ -0,0 +1,234 @@
+sidebar: home_sidebar
+permalink: /ios-release/
+---
+
+During development, you'll test an app using `flutter run` at the command line,
+using the Run and Debug toolbar buttons in IntelliJ, or both. By default,
+Flutter builds a *debug* version of your app.
+
+When you're ready to ship your app to users on the [App Store][appstore] or
+[TestFlight][testflight], you'll need to prepare a *release* version.
+
+This guide provides a step-by-step walkthrough of releasing a typical Flutter
+app to the App Store. For a detailed overview, see the [App Distribution
+Guide][distributionguide].
+
+* TOC Placeholder
+{:toc}
+
+## Preliminaries
+
+Before beginning the process of releasing your app, ensure that it meets
+Apple's [App Review Guidelines][appreview].
+
+If you haven't yet done so, you'll need to [enroll][devprogram_enroll] in the
+[Apple Developer Program][devprogram] in order to publish your app on the App
+Store. You can read more about the various membership options in Apple's
+[Choosing a Membership][devprogram_membership] guide.
+
+## Register your app on iTunes Connect
+
+[iTunes Connect][itunesconnect] is where you'll manage your app's life cycle.
+This is where you'll set your app name and description, add screenshots, set
+pricing, and manage releases to the App Store and TestFlight.
+
+Registering your app involves two steps: registering a unique 'Bundle ID', and
+creating an application record on iTunes Connect.
+
+For a detailed overview of iTunes Connect, see the [iTunes Connect Developer
+Guide][itunesconnect_guide].
+
+### Register a Bundle ID
+
+Every iOS application is associated with a Bundle ID, a unique identifier
+registered with Apple. To register a bundle ID for your app, follow these
+steps:
+
+1. Open the [App IDs][devportal_appids] page of your developer account.
+1. Click '+' to create a new Bundle ID.
+1. Enter an app name and select `Explicit App ID` and enter an ID.
+1. Select the services your app will use, then click `Continue`.
+1. On the next page, confirm the details then click `Register` to register your
+   Bundle ID.
+
+### Create an application record on iTunes Connect
+
+Next, you'll register your app on iTunes Connect:
+
+1. Open [iTunes Connect][itunesconnect_login] in your browser.
+1. On the iTunes Connect landing page, click `My Apps`.
+1. Click '+' in the top-left corner of the `My Apps` page, then select `New
+   App`.
+1. Fill in your app details on the form that appears. In the `Platforms`
+   section, ensure that iOS is checked. Leave tvOS unchecked. Click `Create`.
+1. Navigate to the application details for your app and select `App
+   Information` from the sidebar.
+1. In the `General Information` section, select the Bundle ID you registered in
+   the preceding step.
+
+For a detailed overview, see [Creating an iTunes Connect Record for an
+App][itunesconnect_guide_register].
+
+## Review Xcode project settings
+
+Before you create a build, it's important to ensure that the application
```

Comment:
Good point, this sentence adds no real value over the following paragraph. Deleted.